nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Cj Post Crud

# Description
CRUD For Post and view user posts
Fixes #5 #9 #10 #11
# Testing Instructions
Go to my post and view posts, add new post, in the post details edit the post and delete the post (there are no comments or posttags to check the delete function)
# Checklist:
- [ ] Can Add Post
- [ ] Cand Edit  Post
- [ ] Can View Your Post
- [ ] Cannnot edit or delete another user post
